src/components/UI/Display/Apps/AppList/AppsListPage.tsx

```diff
@@ -105,18 +110,24 @@ const AppsList: React.FC<IAppsListPageProps> = (props) => {
         )}
 
         {!props.appsIsLoading && props.apps.length > 0 && (
-          <List>
-            {props.apps.map((app, i) => (
-              <App
-                key={app.name + i.toString()}
-                name={app.name}
-                appIconURL={app.appIconURL}
-                catalogTitle={app.catalogTitle}
-                catalogIconUrl={app.catalogIconUrl}
-                to={app.to}
-              />
-            ))}
-          </List>
+          <ListContainer>
+            <AppGrid
+              items={props.apps}
+              itemMinHeight={200}
+              itemMinWidth={270}
+              gridGap={20}
+              render={(app: IAppProps, i: number) => (
+                <App
+                  name={app.name}
+                  appIconURL={app.appIconURL}
+                  catalogTitle={app.catalogTitle}
+                  catalogIconUrl={app.catalogIconUrl}
+                  to={app.to}
+                  key={app.name + i.toString()}
```

Comment:
This could actually be the `.to` property, since it is always unique, so we won't need to use the index
```suggestion
                  key={app.to}
```